app/frontend/wallet/cardano-wallet.ts

```diff
@@ -10,10 +8,9 @@ import {MAX_INT32} from './constants'
 import shuffleArray from './helpers/shuffleArray'
 import NamedError from '../helpers/NamedError'
 import {Lovelace} from '../state'
-
 import {computeRequiredTxFee, selectMinimalTxPlan, isUtxoProfitable} from './byron-tx-planner'
-
 import {MaxAmountCalculator} from './max-amount-calculator'
+// eslint-disable-next-line no-unused-vars
 import {TxPlan} from './shelley/build-transaction'
```

Comment:
you can import type directly and save this workaround if you update typescript to the latest version as this feature was added in typescript 3.8: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html